lib/css/modules/_accountSwitcher.scss

```diff
@@ -1,8 +1,10 @@
 #RESAccountSwitcherDropdown {
-	min-width: 110px;
-	width: auto;
 	display: none;
+	height: calc(100vh - 80px);
```

Comment:
This forces the height to always be `calc(100vh - 80px)`. It should use `max-height`.